### PR TITLE
Gsc v1.0

### PR DESCRIPTION
**Pull request by adrpadua**
add genestealer cult basic datasheets

### CODE REVIEW

**Comment by korzxd** [comment]
Nice one! Thanks for contributing another one. I'll get round to reviewing this in the next couple days 👍

File: Genestealer Cult/Readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Units - Standard
+
+* #### Abominant:
+  * **`The Chosen One:`** *Leader Ability* <br> Enable the profile ability `SUSTAINED HITS 1`.
+
+* #### Biophagus:
+  * **`Twisted Science:`** *Leader Ability* <br> Enable the profile ability `LETHAL HITS`.
+
+* #### Magus:
+  * **`Bodyguard:`** *Leader Ability* <br> Enable the profile ability `Feel no pain 4+`.
```
Comment (korzxd) [inline Genestealer Cult/Readme.md:27]: #### Magus
* Change `Bodyguard` to `Spiritual Leader` also change `Feel no pain 4+` to `5+`

File: Genestealer Cult/Readme.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+* #### Patriarch:
+  * **`Might From Beyond:`** *Leader Ability* <br> Enable the profile ability `DEVASTATING WOUNDS`.
```
Comment (korzxd) [inline Genestealer Cult/Readme.md:30]: #### Patriarch
* Missing `Psyker` Keyword.

File: Genestealer Cult/Readme.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+* #### Primus:
+  * **`Cult Demagogue:`** *Leader Ability* <br> Enable the profile ability `Re-roll all possible failed hit rolls (if attacker)`.
+
+* #### Sanctus:
+  * **`Psychic Spoor:`** <br> Enable the profile ability `IGNORES COVER`.
+  * **`Psychic Spoor:`** <br> Enable the profile ability `DEVASTATING WOUNDS`.
```
Comment (korzxd) [inline Genestealer Cult/Readme.md:37]: #### Sanctus
* Add `Stealth` profile ability.

**Review by korzxd — COMMENTED** [review]
Good job on this! Thanks for the help it's mostly just adjustments on profile abilities and the odd keyword here and there, your weapon profiles are spot on by the looks of it along with the unit characteristics. 😄

**Comment by dixhuit** [comment]
Love it. Keep up the splendid work folks :100:

**Comment by adrpadua** [comment]
@korzxd updated!

**Pull request merged**